Index: server/server.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- server/server.js	(revision 2704cfca5c2e3e8dc6d43de565b7894e17a1d330)
+++ server/server.js	(revision )
@@ -11,7 +11,6 @@
 import webpackDevMiddleware from 'webpack-dev-middleware';
 import webpackHotMiddleware from 'webpack-hot-middleware';
 
-
 // Initialize the Express App
 const app = new Express();
 
@@ -78,7 +77,7 @@
 
         ${process.env.NODE_ENV === 'production' ? `<link rel='stylesheet' href='${assetsManifest['/app.css']}' />` : ''}
         <link href='https://fonts.googleapis.com/css?family=Lato:400,300,700' rel='stylesheet' type='text/css'/>
-        <link href='/styles/fonts.css' rel='stylesheet' type='text/css'/>
+        <link href='${assetsManifest['/styles/fonts.css']}' rel='stylesheet' type='text/css'/>
       </head>
       <body>
         <div id="root">${html}</div>
